tools/fastp/fastp.xml

```diff
@@ -0,0 +1,332 @@
+<tool id="fastp" name="fastp" version="0.12.3">
+    <description>Fast all-in-one preprocessing for FastQ files</description>
+    <requirements>
+        <requirement type="package" version="0.12.3">fastp</requirement>
+    </requirements>
+    <command><![CDATA[
```

Comment:
[`<version_command>`](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-version-command)  would be excellent here

Reply:
done